### DISCUSSION TITLE
Tracking Competitor Rankings

### DISCUSSION THREAD

**Discussion by Mitsu66** [towfiqi/serpbear#182] [Ideas]
Hi,

I've been considering how beneficial it would be to also monitor my competitors' rankings for the same keywords using SerpBear. However, if I add my competitors' domains and configure it to check the same keywords with the same settings, it would consume more credits from my scraping API. Despite this, having access to this data could be incredibly valuable.

It would be great if there was a feature in the project allowing us to add competitors and then select them from a dropdown list to view their rankings. This would streamline the process and make the comparison more efficient.

**Comment by dangson92** [comment]
I agree with you, it's really helpful if checking many website for 1 keyword.

**Comment by rsebi** [comment]
This would be a nice addition.

**Comment by syamsulalam** [comment]
this is incredibly important to not burn through scraper credit for several domains which has same keywords
